README.md

```diff
@@ -1,3 +1,7 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+Here is a link to my loom video [Project description video](https://www.loom.com/share/7a461598ef784fe1aa1b03ecf7f3bf13)
+
 # Here is a link to my loom video [Project description video]()
```

Comment:
- [ ] Please combine these two lines to have only one sentence containing the link to your project presentation video. Having two statements referring to one thing is not looking good in your README 🏃 🏃

Reply:
Great. Thanks for the heads up.